### PR TITLE
[New Term for Existing Concept Entry] Pytorch Tensor .size()

### LINKED ISSUE #5058: [Term Entry] PyTorch Tensors .size()
### Reason/inspiration (optional)

Add a new term entry about the `.size()` method for Tensors. Place your file under the file path `docs/content/pytorch/concepts/tensors/terms/size/size.md` 

It should include the following sections:

- A "Syntax" section with a \```pseudo code block
- An "Example" section with a \```py code block
- A "Codebyte Example" section with a \```codebyte/python code block

Please refer to the [concept entry templates](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by maroline-johnson**
Add a new term entry about the .size() method for Tensors.

### Description

I've added a new term entry - .size() - for an existing concept - content/pytorch/concepts/tensors. I created a new folder - size - with a new file - size.md. 

### Issue Solved

Closes #5058

### Type of Change

- Adding a new entry

### Checklist

- [X ] All writings are my own.
- [X ] My entry follows the Codecademy Docs style guide.
- [X ] My changes generate no new warnings.
- [X ] I have performed a self-review of my own writing and code.
- [X ] I have checked my entry and corrected any misspellings.
- [X ] I have made corresponding changes to the documentation if needed.
- [X ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X ] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Review by SaviDahegaonkar — CHANGES_REQUESTED** [review]
Hey @maroline-johnson,
First of all, I must say, the entry is nicely written. I would suggest a very few minor changes to this PR to get it approved for the first review. Please do the suggested changes. Also, I would like to add that you should add a small codebyte example at the end of this entry. Please do it asap and let me know.

Thanks,
Savi

File: content/pytorch/concepts/tensors/size/size.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  - 'Deep Learning'
+  - 'Methods'
+CatalogContent: 
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+  - 'paths/data-science-foundations'
+---
+
+The **`size()`** method returns a `torch-size` object containing the size information of a Pytorch tensor. It is a fundamental function used in PyTorch to obtain the tensor's shape and is specifically used for dynamic operations. You can also access specific dimensions using tuples.
```
Comment (SaviDahegaonkar) [inline content/pytorch/concepts/tensors/size/size.md:19]:
```suggestion
The **`size()`** method returns a `torch.Size` object containing the size information of a Pytorch tensor. It is a fundamental function used in PyTorch to obtain the tensor's shape and is specifically used for dynamic operations. You can also access specific dimensions using tuples. 
```

File: content/pytorch/concepts/tensors/size/size.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+## Syntax
+
+```psuedo
+t.size(dim=None)
+```
+
+- `dim`: (`int`) This optional parameter specifies the dimension for which to retrieve the size. The default value is `None`. 
+    - If `dim` is not provided, the returned value is a `torch.Size`, a subclass of __tuple__. 
+    - If `dim` is specified, the returned value is an __int__ holding the size of the given dimension.
```
Comment (SaviDahegaonkar) [inline content/pytorch/concepts/tensors/size/size.md:29]:
```suggestion
- `dim`: (`int`) This optional parameter specifies the dimension for which to retrieve the size. The default value is `None`. 
    - If `dim` is not provided, the returned value is a `torch.Size`, a subclass of __tuple__. 
    - If `dim` is specified, the returned value is an int holding the size of the given dimension. 
```

File: content/pytorch/concepts/tensors/size/size.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+## Codebyte Example
+
+```codebyte/python
+import torch
+
+t = torch.empty(3, 4, 5)
+
+print(t.size())
+print(t.size(dim=1))
```
Comment (SaviDahegaonkar) [inline content/pytorch/concepts/tensors/size/size.md:59]:
```suggestion
import torch

t = torch.empty(3, 4, 5)

print(t.size())
print(t.size(dim=1))
```
Please add another codebyte example.

**Review by SaviDahegaonkar — CHANGES_REQUESTED** [review]
Hey @maroline-johnson,
The entry is nicely written. I have again suggested a very few changes, please do it asap.

Thanks,
Savi

**Comment by SaviDahegaonkar** [comment]
Hey @maroline-johnson ,
Have you implemented the suggested changes? If yes then let me know so I can review it and approve it.


Thanks,
Savi

**Comment by maroline-johnson** [comment]
@SaviDahegaonkar Yes, I have. You can see them here: [Changes for code review - c1270b2](https://github.com/Codecademy/docs/pull/5088/commits/c1270b2ee8bfcab35fc10606eb35965d4601dd18)

File: content/pytorch/concepts/tensors/size/size.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+  - 'paths/data-science-foundations'
+---
+
+The **`size()`** method returns a `torch.Size` object containing the size information of a Pytorch tensor. It is a fundamental function used in PyTorch to obtain the tensor's shape and is specifically used for dynamic operations. You can also access specific dimensions using tuples. 
+
+## Syntax
+
+```psuedo
+t.size(dim=None)
+```
```
Comment (SaviDahegaonkar) [inline content/pytorch/concepts/tensors/size/size.md:25]:
```suggestion
```pseudo
t.size(dim=None)
```
```

**Review by SaviDahegaonkar — CHANGES_REQUESTED** [review]
Hello @maroline-johnson ,
I had a look at your file suggested a change please make it asap.

Thanks,
Savi

File: content/pytorch/concepts/tensors/size/size.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+  - 'Deep Learning'
+  - 'Methods'
+CatalogContent: 
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+  - 'paths/data-science-foundations'
+---
+
+The **`size()`** method returns a `torch.Size` object containing the size information of a Pytorch tensor. It is a fundamental function used in PyTorch to obtain the tensor's shape and is specifically used for dynamic operations. You can also access specific dimensions using tuples.
```
Comment (SaviDahegaonkar) [inline content/pytorch/concepts/tensors/size/size.md:19]:
```suggestion
The **`size()`** method returns a `torch.Size` object that behaves like a tuple containing the size and information of a PyTorch tensor. This function is a fundamental one used in PyTorch to obtain the tensor's shape. You can also access specific dimensions using index-based access, same as tuples. 
```

**Review by SaviDahegaonkar — APPROVED** [review]
Hey @maroline-johnson ,
LGTM, I hereby approve this PR.

**Review by mamtawardhani — APPROVED** [review]
Thank you for contributing @maroline-johnson 😄 

The entry looks good to be merged! 🚀
